README.md

```diff
@@ -6,32 +6,48 @@
 This project provides a simple UI to visualize the results of [AWS Lambda Power Tuning](https://github.com/alexcasalboni/aws-lambda-power-tuning).
 The UI is a static HTML page that reads data from URL hash.
 
-![Sample Screenshot](sample-screenshot.png?raw=true)
+![Sample Screenshot](docs/sample-screenshot.png?raw=true)
```

Comment:
Are `docs/lambda-analyzer.png` and `docs/sample-with-compare.png` also used somewhere? I see they're new images but can't find any references.

Reply:
Good pickup! I took a second screenshot because the existing one didn't show the Compare button. However, the new one doesn't show the "Balanced cost" option either, so I thought I'd let you decide which to use.